.bowerrc

```diff
@@ -1,4 +1,8 @@
 {
-  "registry": "https://registry.bower.io",
-  "directory": "res/bower_components"
+  "registry": "http://registry.bower.io",
+  "directory": "/tmp/build/bower_modules",
```

Comment:
Even if there is an explanation in the commit message, I'd rather not keep it that way.